tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -4,33 +4,37 @@
 import pandas as pd
 import pytest
 
-from tardis.montecarlo.packet_source import BlackBodySimpleSource
+from tardis.montecarlo.packet_source import BlackBodySimpleSource, BasePacketSource
```

Comment:
I don't think you're using the `BasePacketSource`, right?

Reply:
fixed in https://github.com/tardis-sn/tardis/pull/2407/commits/0c300400d637b427b49fdd7d1f6ee9cc66b8a02b